### ISSUE TITLE
Where themes?)

### ISSUE THREAD

**Issue by SlimRG** [fluidd-core/themes#2]
### Is your feature request related to a problem? Please describe

There no theme description.

### Describe the solution you'd like

Maybe you give an instruction - how to create theme?

**Comment by debemdeboas** [comment]
+1

**Comment by ejcupcake** [comment]
+1

**Comment by Gr3yW0lf2006** [comment]
+1

**Comment by kitOSx** [comment]
+ 1

**Comment by TexAcoon** [comment]
I agree ... "What Themes"?   No instructions no themes .. what good is this in github?
